docs/API/InstrumentFormat.md

```diff
@@ -30,7 +30,11 @@ Instrument format:
         "InstrumentVersion": string,
         "InstrumentFormatVersion" : "v0.0.2-dev",
         "ShortName" : "InstrumentName", /* Required */
-        "LongName"  : "The Human Readable Instrument Name", /* Required */
+        "LongName": {
+          "en-ca": "The Human Readable Instrument Name",
+          "fr-ca": "Bonjour"
```

Comment:
I think the CA part should be capitalized. It looks weird to have "en-ca" and not "en-CA".

For the documentation, it should also be a real translation and not something completely different.

Reply:
suggest: "Instrument Title" / "Titre de l'Instrument"